packages/azure-tracing/README.md

```diff
@@ -68,6 +68,29 @@ registerAzureFunctionHooks(app);
 ...
 ```
 
+### Instrumenting Azure Functions v3 Handlers
+
+For Azure Functions using the v3 programming model, you can use the `withOtelContextFunctionV3` helper function to wrap your handlers and ensure OpenTelemetry context propagation. This function works with the v3 `Context` object structure.
```

Comment:
question: do the same consideration for ESM modules apply? (ie. you still have to set `NODE_OPTIONS` in case)

Reply:
yes, that is the suggested way to integrate the tracing package to have full correlation between requests.

Should I explicit it in another way?